src/data/projects-2019FA.js

```diff
@@ -1,4 +1,12 @@
 const projects2019Fa = [
+  {
+    by: "Tim Cronin",
+    title: "Print\Resist\Project",
```

Comment:
Thanks, @lafesmith! One suggested change:

```suggestion
    title: "Print\\Resist\\Project",
```

A backslash `\` marks the beginning of [escape sequence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#escape_notation) in a string. For example `\n` is a newline character.

The title `Print\Resist\Project` will be parsed with two unintentional escapes: `\R` and `\P`. 

For the backslash to render properly in the title, you'll need to escape it with a backslash before the backslash:`\\`.